Doc/library/pickle.rst

```diff
@@ -494,7 +494,7 @@ What can be pickled and unpickled?
 
 The following types can be pickled:
 
-* ``None``, ``True``, and ``False``;
+* ``None``, ``Ellipsis``, ``True``, and ``False``;
```

Comment:
As we now include more items than the 'obvious' None/True/False, give the group a name, and order by likely relative usage (e.g. put Ellipsis and NotImplemented last)

```suggestion
* built-in constants (``None``, ``True``, ``False``, ``Ellipsis``, and
  ``NotImplemented``);
```